obs-packaging/distros

```diff
@@ -8,6 +8,7 @@ CentOS_7::CentOS:CentOS-7::standard
 Fedora_26::Fedora:26::standard
 Fedora_27::Fedora:27::standard
 Fedora_28::Fedora:28::standard
+Fedora_29::Fedora:29::standard
```

Comment:
I wasn't aware of this file. We'll need to add Debian 9.0 here too.

Reply:
@jcvenegas We should probably remove Fedora 26 and 27 from this list, since they have reached EOL.

Reply:
+1. Raised #279 to deal with that.